scheduler/test/cook/test/mesos/api.clj

```diff
@@ -1589,3 +1590,24 @@
       (is (nil? (:job/name job-entity)))
       (is (= job-uuid (:uuid job-map-for-api)))
       (is (= "cookjob" (:name job-map-for-api))))))
+
+(deftest test-cors
+  (setup :config {:cors-origins ["http://cors.twosigma.com"]})
+  (let [conn (restore-fresh-database! "datomic:mem://test-cors")
+        h (basic-handler conn)]
+    (testing "valid preflight request"
+      (let [{:keys [status headers]} (h {:request-method :options
+                                         :scheme :http
+                                         :uri "/info"
+                                         :headers {"origin" "http://cors.twosigma.com"}})]
```

Comment:
Nitpick: lexicographic order

Reply:
I removed this test since the change is now in components